app/drivers/display/il0323.c

```diff
@@ -0,0 +1,467 @@
+/*
+ * Copyright (c) 2020 PHYTEC Messtechnik GmbH
```

Comment:
This should now include me in the list, e.g.:

```suggestion
 * Copyright (c) 2020 PHYTEC Messtechnik GmbH, Peter Johanson
```

Reply:
done